### DISCUSSION TITLE
Some features worth adding.

### DISCUSSION THREAD

**Discussion by neotoxicfr** [wizarrrr/wizarr#35] [Feature Requests]
Hi,

First of all, thank you for this tool which is super cool.
I thought of maybe more features to make it even better maybe : 
- Adding a link to a discord server.
- Adding a free text / links section maybe, to give some more infos like links to configuration guides, our organizr to see the calendar, discord invitation if the idea above is not ok etc...
- Having the possibility to give the plex download permission to the user, maybe even restrictions (like we would do when adding by hand).

If one day you make a Weblate page for example, I'll happily translate it to french.

**Comment by neotoxicfr** [comment]
More ideas : 
- Custom logo / favicon
- Progression bar for the joining user

**Comment by nagash79** [comment]
Great project!
Here's what immediately came to my mind:

- A checkbox to give invited user access to request system and Discord or not

**Comment by dbadenock** [comment]
Really Really nice tool to simplify the process of added users. I would like to request feature pls.

After creating an Invite admin can enter the emails address of the new user.. This email will be sent from a preset template with the Invite link..

Thanks again for work..

**Comment by oamster** [comment]
Would be awesome to have the option to show or hide overseerr/request on an individual invitation/user basis. Have some users I don't want to know about requests, while others I want to notify.
